app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -13,6 +13,11 @@ def show
       @upload_disabled = idv_session.selfie_check_required &&
                          !idv_session.desktop_selfie_test_mode_enabled?
 
+      @direct_ipp_with_selfie_enabled = IdentityConfig.store.in_person_doc_auth_button_enabled &&
+                                        Idv::InPersonConfig.enabled_for_issuer?(
```

Comment:
I may very much be wrong here so please poke back- vacation brain. 

I am wondering if the new `in_person_doc_auth_button_enabled` can be replaced by the already existing `in_person_proofing_enabled` flag?  If you follow the flow through- if you fail remote and this flag is set to false- you will only be asked to retry remote- you will not have the option to move into IPP.  If the flag is set to true (and you fail remote)- you will have the option to retry remote or start IPP. It seems like when we want the same logic here. Is there a situation in which the flags could be different?

Reply:
@gina-yamada , that `in_person_proofing_enabled` flag is already considered in `Idv::InPersonConfig.enabled_for_issuer?`.

Reply:
This raises the question should we push `in_person_doc_auth_button_enabled` down there too?